doc/source/index.rst

```diff
@@ -65,6 +65,20 @@ To further enhance data discovery, we have developed the QUEST module to facilit
         :text: Software Docs
         :classes: stretched-link btn-outline-primary btn-block
 
+    ---
+    :img-top: https://cdn-icons-png.flaticon.com/256/9585/9585915.png
+
+    **QUEST**
+    ^^^^^^^^^^^^^^
+
+    Query, Unify, Explore SpatioTemporal (QUEST) is a module that extends icepyx functionality to other
+    datasets.
+
+    .. link-button:: `QUEST_Finding_Argo_IS2_data <https://icepyx.readthedocs.io/en/latest/example_notebooks/QUEST_argo_data_access.html>`_
+        :type: url
+        :text: Start your QUEST!
+        :classes: stretched-link btn-outline-primary btn-block
```

Comment:
```suggestion
    .. button-ref:: /example_notebooks/QUEST_argo_data_access.ipynb
        :class: stretched-link btn-outline-primary btn-block
        
        Start your QUEST!
```

sphinx-panels ([dead project](https://sphinx-panels.readthedocs.io/en/latest/)) is replaced by sphinx-design on main. [Minor changes required](https://sphinx-design.readthedocs.io/en/latest/get_started.html#migrating-from-sphinx-panels). 

I'm not sure if the link will work exactly as written, but what I'm trying to do is replace an absolute URL with generating a URL from a ref.